src/styles/blocks/clock.scss

```
@@ -0,0 +1,49 @@
+.clock {
+  @include setSize;
+  border-radius: 50%;
```

```suggestion
  @include circle($size);
```

Combine 3 props to create a universal mixin. Reuse it with `dot`.